backend/verify_llm_fixes.py

```
@@ -0,0 +1,130 @@
+        os.environ['LLM_PROVIDER'] = 'groq'
+        
+        # Force reload of settings
+        import importlib
+        from app import config
+        importlib.reload(config)
+        
+        from app.llm.manager import LLMManager
+        manager = LLMManager()
+        
+        # This should use Groq's default model, not crash
+        llm = manager.get_llm("groq")
```

Ruff (0.14.13)

Local variable `llm` is assigned to but never used

Remove assignment to unused variable `llm`

(F841)